src/request/sign-transaction/SignTransactionApi.js

```diff
@@ -39,7 +55,7 @@ class SignTransactionApi extends TopLevelApi {
     /**
      * Checks that the given layout is valid
      * @param {any} layout
```

Comment:
Not in this PR, but this should probably be 
```suggestion
      * @param {unknown} layout
```

Reply:
Why would that be better than `any`?
Isn't `any` appropriate here?
The method can handle `any`thing and returns the parsed layout if the input could actually be parsed as a layout.

Reply:
`unknown` is less permissive than `any`. 
As an example
```(x: any) => { console.log(x.test); }```
is valid for the typechecker, while
```(x: unknown) => {console.log(x.test); }```
is not.

Therefore I think it would be better for the parser to assume `unknown` than `any`, generally speaking.

Reply:
I see.
But I'd say, that should be part of a separate PR that makes this change across the whole project.

Reply:
I agree.